app/components/buttons/play-button-main.jsx

```
@@ -29,7 +30,7 @@
         onClick()
       }} {...props}>
           <div className={`state-layer relative h-8 w-8 md:p-3 md:h-12 md:w-12 rounded-full flex items-center justify-center ${className} md:bg-[--md-sys-color-primary] md:text-[--md-sys-color-on-primary]`}>
-              <md-ripple></md-ripple>
+              <md-ripple className={`${props.disabled ? "hidden" : ""}`}></md-ripple>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)